CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+# 1.4.1
+> Published 23 Sep 2020
+
+* OkHttp: Can't reuse same HttpRequestBuilder for different network clients (KTOR-949)
```

Comment:
Will issues' ID be converted to Youtrack links?

Reply:
I will try to make this